config/application.yml.default

```diff
@@ -117,8 +117,8 @@ idv_contact_url: https://www.example.com
 idv_contact_phone_number: (844) 555-5555
 idv_max_attempts: 5
 idv_min_age_years: 13
-idv_acuant_sdk_version_default: '11.7.1'
-idv_acuant_sdk_version_alternate: '11.7.0'
+idv_acuant_sdk_version_default: '11.8.1'
```

Comment:
Should this be `11.8.0`? We don't have files for a `11.8.1`.

https://github.com/18F/identity-idp/tree/main/public/acuant

Reply:
Wow, bad typo!

Reply:
Kinda curious if the specs would have caught that. They didn't run for the last commit since there was a missing changelog.

Reply:
I'm almost positive they would not have. I think we should add a check to make sure that anything in the config matches to a public dir reference to a real acuant version. Could be a simple spec -- what do you think?

Reply:
> I'm almost positive they would not have. I think we should add a check to make sure that anything in the config matches to a public dir reference to a real acuant version. Could be a simple spec -- what do you think?

If you're willing to give it a go, I think that'd be a nice thing to have 👍

Reply:
Alright it's a simple test, but I added it in https://github.com/18F/identity-idp/pull/7678/commits/a9be5de0ab86c9a70cd4f400b034ab47d21bf69d

Reply:
> Alright it's a simple test, but I added it in [4bc154f](https://github.com/18F/identity-idp/commit/4bc154fabd074e309d070c906a211b616d7d29ec)

LGTM 👍